BackEndApp/deployments/views.py

```diff
@@ -0,0 +1,260 @@
+from genericpath import isdir
+from django.shortcuts import render
+from django.http import JsonResponse, HttpResponse
+from rest_framework.decorators import api_view
+import os
+import git
+import shutil
+
+from authv1.store import Store
+from authv1.models import User
+from authv1.decorators import is_authenticated
+
+from dlmml.parser import *
+from dotenv import load_dotenv
+
+from .utils import remove_dir, edit_flask_app, zip_flask_app
+
+load_dotenv()
+
+
+@api_view(["POST"])
+@is_authenticated
+def local_deploy(request):
+    """
+    Endpoint for locally deploying user model.
+    Inputs:
+    -------
+    request:
+        Requires: username, project_id, model_categories & model_download_type in request data
+                  token in request header
+    Returns:
+    --------
+    response: HttpResponse (Required Download)
+              (Yes, not JsonResponse like other views)
+    """
+
+    try:
```

Comment:
Also, since it's a new Django App and won't need any change to the code that is not a part of the PR, make use of the logger object and add a few logs wherever necessary. As it makes it easier to debug during testing. 

Sample code: 
```python
import logging
logger = logging.getlogger(__name__)
```

Reply:
> It would be better if the entire function definition is not placed inside the `try` block. Rather place `try-except` blocks wherever you need them.

Yes i'll try to scope the exception handling to the required parts!

Reply:
> Also, since it's a new Django App and won't need any change to the code that is not a part of the PR, make use of the logger object and add a few logs wherever necessary. As it makes it easier to debug during testing.
> 
> Sample code:
> 
> ```python
> import logging
> logger = logging.getlogger(__name__)
> ```

Ahh yes will do this as well! (P.s something new to try out again 😆)